backend/src/process.py

```diff
@@ -155,9 +159,38 @@ async def __process(self, node: UsableData) -> Any:
             # Run the node and pass in inputs as args
             run_func = functools.partial(node_instance.run, *enforced_inputs)
             output = await self.loop.run_in_executor(None, run_func)
+            node_outputs = node_instance.get_outputs()
+            broadcast_data = dict()
+            if len(node_outputs) > 0:
+                if len(node_outputs) == 1:
+                    try:
+                        broadcast_data[0] = node_outputs[0].broadcast(output)
+                    except NotImplementedError:
+                        pass
```

Comment:
Please don't do exception-driven control flow.

How about this: `broadcast` is allowed to return `None` as a "nothing to broadcast" state. This would make for a very natural default implementation.